Cython/Compiler/PyrexTypes.py

```diff
@@ -2696,6 +2696,12 @@ def from_py_call_code(self, source_code, result_code, error_pos, code,
             source_code, result_code, self.size)
         return code.error_goto_if_neg(call_code, error_pos)
 
+    def error_condition(self, result_code):
+        # It isn't possible to use CArrays return type so the error_condition
+        # is irrelevant. Returning a Falsey value does avoid an error when getting
+        # from_py_call_code from a typedef
```

Comment:
From a web search, "falsy" seems a somewhat more common spelling than "falsey", although both seem to be used.
```suggestion
        # It isn't possible to use CArrays as return type so the error_condition
        # is irrelevant. Returning a falsy value does avoid an error when getting
        # from_py_call_code from a typedef.
```